### ISSUE TITLE
What do we think of vintage fonts?

### ISSUE THREAD

**Issue by FuriouslyCurious** [ProgrammingFonts/ProgrammingFonts#10]
There is a vintage mono-fonts collection here: 
https://int10h.org/oldschool-pc-fonts/readme/

Do these vintage fonts fit the spirit of this repo?

**Comment by ifeegoo** [comment]
@FuriouslyCurious Yeah,this is really good!All mono fonts fit the spirit of this repository.

**Comment by ifeegoo** [comment]
@FuriouslyCurious Please tell me your email address,so I can contact with you privately.
